### PR TITLE
[Bugfix] Restore sanity test

### PR DESCRIPTION
**Pull request by Gamenot**
- This restores the `sanity-test` for now
- This also splits up the scenarios into groups

There is still something to comment about splitting up the main test.

### CODE REVIEW

File: smarts/sstudio/tests/test_generate.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ def fake_map_builder(map_spec: MapSpec) -> bool:
 
 
 def test_generate_od_map():
-    scenario_root = "scenarios/od_4lane"
+    scenario_root = "scenarios/smarts/od_4lane"
```
Comment (Adaickalavan) [inline smarts/sstudio/tests/test_generate.py:133]: Is it `smarts` or `open_drive`?
```suggestion
    scenario_root = "scenarios/open_drive/od_4lane"
```

File: scenarios/smarts/tests/same_location/map.net.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 <configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://sumo.dlr.de/xsd/netconvertConfiguration.xsd">
 
     <output>
-        <output-file value="/home/dev/Desktop/repos/SMARTS/scenarios/tests/congestion/map.net.xml"/>
+        <output-file value="/home/dev/Desktop/repos/SMARTS/scenarios/smarts/tests/congestion/map.net.xml"/>
```
Comment (Adaickalavan) [inline scenarios/smarts/tests/same_location/map.net.xml:7]: Where is this folder `SMARTS/scenarios/smarts/tests/congestion/` ?

File: smarts/core/tests/test_map.py [hunk 3]
```diff
@@ -36,17 +36,17 @@
...
 @pytest.fixture
 def sumo_scenario():
-    return Scenario(scenario_root="scenarios/intersections/4lane")
+    return Scenario(scenario_root="scenarios/smarts/intersections/4lane")
 
 
 @pytest.fixture
 def opendrive_scenario_4lane():
-    return Scenario(scenario_root="scenarios/od_4lane")
+    return Scenario(scenario_root="scenarios/smarts/od_4lane")
```
Comment (Adaickalavan) [inline smarts/core/tests/test_map.py:44]:
```suggestion
    return Scenario(scenario_root="scenarios/open_drive/od_4lane")
```

File: scenarios/smarts/tests/same_location/map.net.xml (hunk 2 above)
Comment (Gamenot) [reply scenarios/smarts/tests/same_location/map.net.xml]: This was the original output file, it is not an issue but I have corrected it.

**Comment by Gamenot** [comment]
> The scenarios were divided into `open_drive`, `smarts`, `waymo`, `NGSIM`, `INTERACTION` folders. Is it better to call the `smarts` folder as `sumo` folder instead?

Yes, I realized that this morning as well. I have fixed it.

File: scenarios/NGSIM/us101/map.net.xml [hunk 4]
```diff
@@ -4,11 +4,11 @@
 <configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://sumo.dlr.de/xsd/netconvertConfiguration.xsd">
 
     <input>
-        <sumo-net-file value="scenarios/NGSIM/us101/map.net.xml"/>
+        <sumo-net-file value="scenarios/sumo/NGSIM/us101/map.net.xml"/>
     </input>
 
     <output>
-        <output-file value="scenarios/NGSIM/us101/map.net.xml"/>
+        <output-file value="scenarios/sumo/NGSIM/us101/map.net.xml"/>
```
Comment (Adaickalavan) [inline scenarios/NGSIM/us101/map.net.xml:11]:
```suggestion
        <output-file value="scenarios/NGSIM/us101/map.net.xml"/>
```

File: Makefile [hunk 5]
```diff
@@ -73,6 +73,15 @@ run: build-scenario
 build-all-scenarios:
 	scl scenario build-all scenarios
 
+.PHONY: build-smarts-scenarios
+build-smarts-scenarios:
```
Comment (Adaickalavan) [inline Makefile:77]: Perhaps?
```suggestion
.PHONY: build-sumo-scenarios
build-sumo-scenarios:
```

**Pull request merged**